src/keymap.c

```diff
@@ -75,17 +75,18 @@ Lisp_Object control_x_map;	/* The keymap used for globally bound
 				   bindings when spaces are not encouraged
 				   in the minibuf.  */
 
+/* Hash table used to cache a reverse-map to speed up calls to where-is.  */
+Lisp_Object where_is_cache;
+
```

Comment:
Need protection from GC, see below.

Reply:
Noted.

Reply:
This creates another `where_is_cache` and `where_is_cache_keymaps`.

Reply:
Oh. When i removed this before it complained that the variable is not declared. How do I expose the protected variable from Rust to C?

Reply:
You can't, it is static. You could export a function from Rust to access them. Or you could port the rest of the functions that use the variables.

Reply:
I see, that makes sense. I'll see which option is more viable for me at the moment. Thanks!